client/my-sites/plan-features-2023-grid/components/billing-timeframe.tsx

```diff
@@ -71,16 +72,32 @@ function usePerMonthDescription( {
 				: null;
 
 		if ( fullTermPriceText ) {
+			const isEnglishLocale = currentLocale === 'en' || currentLocale?.startsWith( 'en-' );
+
 			if ( PLAN_ANNUAL_PERIOD === billingPeriod ) {
-				return translate( 'per month, %(fullTermPriceText)s billed annually', {
-					args: { fullTermPriceText },
-				} );
+				// TODO: Remove check once text is translated
+				return isEnglishLocale ||
+					i18n.hasTranslation( 'per month, %(fullTermPriceText)s billed annually, Excl. Taxes' )
```

Comment:
I think we shouldn't do this because there can be a buggy state where ANNUAL will show the `Excl tax` translation and BIENNIAL will not have `Excl. taxes` at all. We should probably string freeze this PR until both texts are translated. 

If we need to race to show this change, then we should probably check both states and show the two translation **_IFF_** both translations have been completed.

Reply:
> I think we shouldn't do this because there can be a buggy state where ANNUAL will show the Excl tax translation and BIENNIAL will not have Excl. taxes at all.

This is a good point 👍 

> If we need to race to show this change, then we should probably check both states and show the two translation IFF both translations have been completed.

Because this PR has to do with potential legal compliance, my intuition says to get these string translations out sooner rather than later. 

"[EU consumer protections](https://href.li/?https://europa.eu/youreurope/citizens/consumers/unfair-treatment/unfair-pricing/index_en.htm) forbid this practice. Beyond legality, I would argue that it is also contrary to standard practices and consumer expectations, consumers who will be put off by this sort of practice and are more likely to turn their backs on WordPress.com." ( pdgrnI-27y-p2 ).

Since I'm newer on the team though, I'm happy to defer to the expertise of you @jdc91 and @aneeshd16

Reply:
The urgency makes sense @jeyip 👍 As long as we guard against the edge case 😸

Reply:
Added a guard against the edge case